slack-base/src/main/java/com/hubspot/slack/client/models/blocks/InputIF.java

```diff
@@ -0,0 +1,35 @@
+package com.hubspot.slack.client.models.blocks;
+
+import java.util.Optional;
+
+import org.immutables.value.Value;
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.blocks.elements.BlockElement;
+import com.hubspot.slack.client.models.blocks.objects.Text;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface InputIF extends Block {
+  String TYPE = "input";
+
+  @Override
+  @Value.Default
+  default String getType() {
+    return TYPE;
+  }
+
+  Text getLabel();
+
+  BlockElement getElement();
+
+  Optional<Text> getHint();
+
+  @JsonProperty("optional")
+  Optional<Boolean> isOptional();
```

Comment:
An `Optional<Boolean>` to tell if it's optional?!

Reply:
It's optionally optional!